README.md

```diff
@@ -118,11 +112,15 @@
+* **[LightRAG](https://github.com/HKUDS/LightRAG)**: "LightRAG: Simple and Fast Retrieval-Augmented Generation" ![Stars](https://img.shields.io/github/stars/HKUDS/LightRAG.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/HKUDS/LightRAG?color=green)  ![LastCommit](https://img.shields.io/github/last-commit/HKUDS/LightRAG?color=green)
+* **[LlamaIndex](https://github.com/run-llama/llama_index)**: LlamaIndex is the leading framework for building LLM-powered agents over your data. ![Stars](https://img.shields.io/github/stars/run-llama/llama_index.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/run-llama/llama_index?color=green) ![LastCommit](https://img.shields.io/github/last-commit/run-llama/llama_index?color=green)
 * **[MetaGPT](https://github.com/geekan/MetaGPT)**: 🌟 The Multi-Agent Framework: First AI Software Company, Towards Natural Language Programming. ![Stars](https://img.shields.io/github/stars/geekan/metagpt.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/geekan/metagpt?color=green) ![LastCommit](https://img.shields.io/github/last-commit/geekan/metagpt?color=green)
 * **[OpenAI Agents SDK](https://github.com/openai/openai-agents-python)**: A lightweight, powerful framework for multi-agent workflows. ![Stars](https://img.shields.io/github/stars/openai/openai-agents-python.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/openai/openai-agents-python?color=green) ![LastCommit](https://img.shields.io/github/last-commit/openai/openai-agents-python?color=green)
 * **[OpenManus](https://github.com/mannaandpoem/OpenManus)**: No fortress, purely open ground. OpenManus is Coming. ![Stars](https://img.shields.io/github/stars/mannaandpoem/openmanus.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/mannaandpoem/openmanus?color=green) ![LastCommit](https://img.shields.io/github/last-commit/mannaandpoem/openmanus?color=green)
 * **[PydanticAI](https://github.com/pydantic/pydantic-ai)**: Agent Framework / shim to use Pydantic with LLMs. ![Stars](https://img.shields.io/github/stars/pydantic/pydantic-ai.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/pydantic/pydantic-ai?color=green) ![LastCommit](https://img.shields.io/github/last-commit/pydantic/pydantic-ai?color=green)
+* **[Semantic Kernel](https://github.com/microsoft/semantic-kernel)**: An open-source integration framework for integrating LLMs into your applications, featuring plugin integration, memory management, planners, and multi-modal capabilities. ![Stars](https://img.shields.io/github/stars/microsoft/semantic-kernel.svg?style=flat&color=green) ![Contributors](https://img.shields.io/github/contributors/microsoft/semantic-kernel?color=green) ![LastCommit](https://img.shields.io/github/last-commit/microsoft/semantic-kernel?color=green)
```

Comment:
Could you update the bio of this project? It's updated.

Reply:
Updated